.gitignore

```diff
@@ -1 +1 @@
-*.ics
+**.ics
```

Comment:
Because I moved it to a folder that isn't in the root, and I forgot how ignore patterns work :stuck_out_tongue: